Resources/doc/reference/configuration/proxy-client.rst

```diff
@@ -323,6 +323,41 @@ the `AWS Async documentation_`. It can not be used with the ``client`` option.
 Service identifier of a `AsyncAws\CloudFront\CloudFrontClient` client. More information is available on the
 `AWS Async documentation_`. It can not be used with the ``configuration`` option.
 
+.. _configuration_fastly_proxy_client:
+
+Fastly
+----------
+.. code-block:: yaml
+
+    # config/packages/fos_http_cache.yaml
+    fos_http_cache:
+        proxy_client:
+            fastly:
+                service_identifier: '<my-service-identifier>'
+                authentication_token: '<my-authentication-token>'
+                soft_purge: true
+
+``service_identifier``
+"""""""""""""""""""
+
+**type**: ``string``
+
+Identifier for the Fastly Service you want to purge the cache for.
+
+``authentication_token``
+"""""""""""""""""
+
+**type**: ``string``
+
+Authentication token (API Token) which can be created in the profile section of your account
+
+``soft_purge``
+"""""""""""""""""
+
+**type**: ``boolean`` **default**: ``true``
+
+Boolean for doing soft purges or not on tag & URL purging. Soft purges expires the cache unlike hard purge (removal), and allow grace/stale handling within Fastly VCL.
```

Comment:
how about:

```suggestion
Whether to expire the cache (true) or whether to directly remove cache entries. Soft purge allows grace/stale handling.
```

(the grace/stale can also be controlled with the cache-control header by the application.)

Reply:
That is copied from the (FOSHttpCache documentation)[https://foshttpcache.readthedocs.io/en/latest/proxy-clients.html#fastly-client]. I think it would make sense to just copy the description?

Reply:
fair enough.